front/src/modules/companies/components/CompanyBoardCard.tsx

```diff
@@ -135,11 +176,15 @@ export const CompanyBoardCard = () => {
     </StyledFieldContainer>
   );
 
+  const OnMouseLeaveBoard = () => {
+    setIsCardInCompactView(true);
+  };
+
   return (
     <StyledBoardCardWrapper>
       <StyledBoardCard
         selected={currentCardSelected}
-        onClick={() => setCurrentCardSelected(!currentCardSelected)}
```

Comment:
I'm surprise we remove this line, not needed anymore?

Reply:
if we don't remove it then the Card get selected when we click on the eye icon in compact mode for expend mode.

is this okay to get selected?? or should i do somethign else?